docs/source/notebooks/wps_resolve_rules_demo.ipynb

```diff
@@ -153,6 +153,55 @@
     "    )"
```

Comment:
<p>Could we move this import statement to the top with the others?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/sandpiper/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Ya sure, I was just trying to be really clear about which library they come from, but I mention it in the markdown so it's probably fine